### ISSUE TITLE
Can't locate path: <.../vendor/binarytorch/larecipe/publishable/config/LaRecipe.php>

### ISSUE THREAD

**Issue by andresgcarmona** [saleem-hadad/larecipe#26]
After running `php artisan larecipe:install` you get this error. It seams that either the filename is wrong or the code in the ServiceProvider has a typo:

```
$publishable = [
    'larecipe_config' => [
        **"{$publishablePath}/config/LaRecipe.php"** => config_path('larecipe.php'),
    ],
    'larecipe_assets' => [
        "{$publishablePath}/assets/" => public_path('vendor/binarytorch/larecipe/assets'),
        "{$publishablePath}/fonts/"  => public_path('fonts'),
    ],
];
```

**Comment by saleem-hadad** [comment]
Hi @andresgcarmona 
Could you please provide the error message?

**Comment by saleem-hadad** [comment]
Thank you @andresgcarmona 
I remembered now that this issue was already solved by one PR [#19](https://github.com/saleem-hadad/larecipe/pull/19/files), the fix will be delivered in the next version.
